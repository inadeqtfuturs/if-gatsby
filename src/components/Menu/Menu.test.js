import React from 'react';
import { Menu } from '@components';
import ThemeRender from '@utils/testing';

const mockItems = [
  { label: 'home', to: '/' },
  {
    label: 'nest',
    to: '/',
    subItems: [
      { label: 'one', to: '/' },
      { label: 'two', to: '/' }
    ]
  }
];

describe('Menu', () => {
  it('renders correctly', () => {
    const tree = ThemeRender(<Menu menuItems={mockItems} />);
    expect(tree).toMatchSnapshot();
  });
});
