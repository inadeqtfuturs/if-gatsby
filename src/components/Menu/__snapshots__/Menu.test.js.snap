// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu renders correctly 1`] = `
Array [
  <nav
    className="sc-Axmtr hZPiec"
    open={false}
  >
    <li
      className="sc-AxheI fBXKxa"
    >
      <a
        className="sc-AxgMl kuYXIx"
        href="/"
      >
        home
      </a>
    </li>
    <li
      className="sc-AxheI fBXKxa"
    >
      <a
        className="sc-AxgMl kuYXIx"
        href="/"
      >
        nest
         
        <svg
          className="sc-AxhUy hTVZBe"
          fill="currentColor"
          height="1em"
          stroke="currentColor"
          strokeWidth="0"
          style={
            Object {
              "color": undefined,
            }
          }
          viewBox="0 0 448 512"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M207.029 381.476L12.686 187.132c-9.373-9.373-9.373-24.569 0-33.941l22.667-22.667c9.357-9.357 24.522-9.375 33.901-.04L224 284.505l154.745-154.021c9.379-9.335 24.544-9.317 33.901.04l22.667 22.667c9.373 9.373 9.373 24.569 0 33.941L240.971 381.476c-9.373 9.372-24.569 9.372-33.942 0z"
          />
        </svg>
      </a>
      <ul
        className="sc-AxhCb brPVnz"
      >
        <li
          className="sc-AxheI fBXKxa"
        >
          <a
            className="sc-AxgMl kuYXIx"
            href="/"
          >
            one
          </a>
        </li>
        <li
          className="sc-AxheI fBXKxa"
        >
          <a
            className="sc-AxgMl kuYXIx"
            href="/"
          >
            two
          </a>
        </li>
      </ul>
    </li>
  </nav>,
  <button
    className="sc-AxmLO efJKYl"
    onClick={[Function]}
  >
    <svg
      fill="currentColor"
      height={24}
      size={24}
      stroke="currentColor"
      strokeWidth="0"
      style={
        Object {
          "color": undefined,
        }
      }
      viewBox="0 0 448 512"
      width={24}
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M16 132h416c8.837 0 16-7.163 16-16V76c0-8.837-7.163-16-16-16H16C7.163 60 0 67.163 0 76v40c0 8.837 7.163 16 16 16zm0 160h416c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H16c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h416c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H16c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16z"
      />
    </svg>
  </button>,
]
`;
